spark-lint/src/test/kotlin/com/adevinta/spark/lint/UnknownStringResourceAnnotationsDetectorTest.kt

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2024 Adevinta
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.adevinta.spark.lint
+
+import com.adevinta.spark.lint.UnknownStringResourceAnnotationsDetector.Companion.UNKNOWN_ANNOTATION_NAME_ISSUE
+import com.adevinta.spark.lint.UnknownStringResourceAnnotationsDetector.Companion.UNKNOWN_ANNOTATION_VALUE_ISSUE
+import com.android.tools.lint.checks.infrastructure.LintDetectorTest
+import com.android.tools.lint.detector.api.Detector
+import com.android.tools.lint.detector.api.Issue
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@RunWith(JUnit4::class)
+public class UnknownStringResourceAnnotationsDetectorTest : LintDetectorTest() {
```

Comment:
The existing `MaterialComposableUsageDetectorTest` has the `public` visibility modifier on its class and methods, so I also added it for `UnknownStringResourceAnnotationsDetectorTest` but I'm not sure we actually need it?

Reply:
It seems like our `explicitApi` is not configured (maybe on purpose?) on test sources.